Future/tests/testthat/test_energy.R

```diff
@@ -0,0 +1,56 @@
+testthat::test_that(desc = "energy_value_of_the_product_per_100 works!", {
+
+  result <- energy_value_of_the_product_per_100(2.7, 1.5, 16)
+  expect <- 88.3
+  testthat::expect_equal(result, expect)
+
+  result <- energy_value_of_the_product_per_100(2.7, 1.5, 16, 2)
+  testthat::expect_equal(result, 84.3)
+})
+
+
+testthat::test_that(desc = "energy_value_of_the_product_per_100 has expected error messeges", {
+
+  testthat::expect_error(energy_value_of_the_product_per_100("2.7", 1.5, 16), "protein must be numeric")
+  testthat::expect_error(energy_value_of_the_product_per_100(2.7, "1.5", 16), "fat must be numeric")
+  testthat::expect_error(energy_value_of_the_product_per_100(2.7, 1.5, "16"), "carbohydrates must be numeric")
+  testthat::expect_error(energy_value_of_the_product_per_100("2.7", 1.5, 16, 2), "protein must be numeric")
```

Comment:
These tests are not necessary (duplicated in 14-16). But we cen keep them

Reply:
ok